ihatemoney/utils.py

```diff
@@ -278,3 +279,71 @@ def coerce(cls, item):
 
     def __str__(self):
         return str(self.value)
+
+
+def em_surround(string, regex_escape=False):
+    # Needed since we're going to assume this is safe later in order to render
+    # the <em> tag we're adding
+    string = escape(string)
+
+    if regex_escape:
+        return r'<em class="font-italic">%s<\/em>' % string
+    else:
+        return '<em class="font-italic">%s</em>' % string
+
+
+def localize_list(list, surround_with_em=True):
+    """
+    Localize a list, optionally surrounding each item in <em> tags.
+
+    Uses the appropriate joining character, oxford comma behavior, and handles
+    1, and 2 object lists, all according to localizable behavior.
+
+    Examples (using en locale):
+        >>> localize_list([1,2,3,4,5], False)
+        1, 2, 3, 4, and 5
+
+        >>> localize_list([1,2], False)
+        1 and 2
+
+    Based on the LUA example from:
+    https://stackoverflow.com/a/58033018
+
+    :param list: The list of objects to localize by a call to str()
+    :param surround_with_em: Optionally surround each object with <em> tags
+    :return: A locally formatted list of objects
+    """
+    list = list[:]
+    one = _("{single_object}")
+    two = _("{dual_object_0} and {dual_object_1}")
+    start = _("{start_object}, {next_object}")
+    middle = _("{previous_object}, {next_object}")
+    end = _("{previous_object}, and {end_object}")
+
+    item_wrapper = em_surround
+    # fmt: off
+    if not surround_with_em:
+        def item_wrapper(x):
+            return x
+    # fmt: on
```

Comment:
I know one-liner are sometimes cryptic, but here, I think it will simplify code.
```suggestion
    item_wrapper = em_surround if surround_with_em else lambda x: x
```